### DISCUSSION TITLE
GitHub Feature Request - custom notifications folders and more swipe options

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [community/community#18623] [Mobile]
I get quite a few notifications but once I read them they "disappear". Afterwards I have to remember that I read them and dig around in my archive for them (the saved option is nice but that can get a bit crowded sometimes - I would love to have a folder for every project I am working on). It would be great if I could save pertinent notifications in custom folders perhaps by swiping hard right or left (currently you can swipe left and right and set those to correspond to specific actions). 

Spark email (iOS version not sure about Android) offers a hard (long) swipe left and hard (long) swipe right that allows custom actions for emails. In effect this gives the user 4 options on what do with their emails. For example I have hard (long) swipe left set to move an email into my Crypto folder. 

Functionality like that but in the GitHub app would be awesome I think.

**Comment by eliperkins** [comment]
Hey @Isaac-Tait! Thanks for this feedback. I'd love to get some more info from ya to help improve the notifications experience on mobile.

> I get quite a few notifications but once I read them they "disappear". Afterwards I have to remember that I read them and dig around in my archive for them 

Are you using the "Unread" filter for your inbox? This will cause the notifications to disappear after reading them. Otherwise, notifications shouldn't disappear out from underneath you.

![unread](https://user-images.githubusercontent.com/1051453/174117352-60544f70-7392-449d-b1d0-d3382cc9e6bb.jpg)

If you're not using the "Unread" filter, let me know and we can look into what's causing this behavior.

Also, if the default swipe actions for Notifications don't fit your needs, GitHub Mobile does offer customization of leading and trailing swipes within Profile > Settings > Notifications > Swipe Options:

![swipe-options](https://user-images.githubusercontent.com/1051453/174117609-ddcbc158-76c0-45e6-afe9-783572fc930e.jpg)

I've jotted down your notes about introducing more swipe options for different length swipes in our own internal tracking issues. I agree that Spark (and the old Mailbox.app for iOS!) both make taking action on many items like this quite easy.

Thanks again for your feedback! Every bit of feedback helps us make a better mobile app ❤️
